cpp/src_prims/score/scores.h

```diff
@@ -78,6 +78,122 @@ namespace MLCommon {
 
             return 1.0 - sse/ssto;
         }
+
+        /**
+          * @brief Compute accuracy of predictions. Useful for classification.
+          * @tparam math_t: data type for predictions (e.g., int for classification)
+          * @param[in] predictions: array of predictions (GPU pointer).
+          * @param[in] ref_predictions: array of reference (ground-truth) predictions (GPU pointer).
+          * @param[in] n: number of elements in each of predictions, ref_predictions.
+          * @param[in] d_alloc: device allocator.
+          * @param[in] stream: cuda stream.
+          * @return: Accuracy score in [0, 1]; higher is better.
+          */
+        template<typename math_t>
+        float accuracy_score(const math_t * predictions, const math_t * ref_predictions, int n,
```

Comment:
Please add unit-tests for this.

Reply:
Will do! Shall I add them under `test/prims/score.cu` or add a file per metric?

Reply:
Yes, for now, please add it inside score.cu. Once the namespace-related confusion has been sorted out, I'll take care of the cleanup business.

Reply:
Done.